### PR TITLE
Add new folder (kannada_kali) to kannada

### PR DESCRIPTION
**Pull request by swatilk**
Please review this, @casey-collab

### CODE REVIEW

File: kannada/kannada_kali/kannada_kali.md [hunk 1]
```diff
@@ -0,0 +1 @@
+ಕನ್ನಡ ಕಲಿಯಲು [ಇದು](https://play.google.com/store/apps/details?id=com.pgjpb.learnkannada&hl=en)  ಒಳ್ಳೆಯ ಸಾಧನ.
```
Comment (casey-collab) [inline kannada/kannada_kali/kannada_kali.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by swatilk** [comment]
Please review this, @casey-collab

File: kannada/soochike.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ಇದು ಕನ್ನಡ ಬರಹಗಳ ಸೂಚಿಕೆ.
 
 ಲೇಖನಗಳು:
-1. ರೋಮಾಂಚನವೀ ಕನ್ನಡ! ಓದಲು, [ಇಲ್ಲಿ ಒತ್ತಿ](romaanchanavee_kannada/romaanchanavee_kannada.md).
\ No newline at end of file
+1. ರೋಮಾಂಚನವೀ ಕನ್ನಡ! ಓದಲು, [ಇಲ್ಲಿ ಒತ್ತಿ](romaanchanavee_kannada/romaanchanavee_kannada.md).
+2. ಕನ್ನಡ ಕಲಿಯಲು [ಇಲ್ಲಿ](kannada_kali/kannada_kali.md).
```
Comment (casey-collab) [inline kannada/soochike.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by swatilk** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**